backend/prisma/seed/config.seed.ts

```diff
@@ -51,6 +51,11 @@ const configVariables: ConfigVariables = {
       defaultValue: "0",
       secret: false,
     },
+    shareIdLength: {
+      type: "number",
+      defaultValue: "16",
```

Comment:
```suggestion
      defaultValue: "8",
```

I would set the default to `8`.

Reply:
Done :heavy_check_mark: